ui/app/src/components/PreviewComponentsPanel/PreviewComponentsPanel.tsx

```diff
@@ -419,7 +419,7 @@ export function PreviewComponentsPanel() {
 						<Avatar component="div" sx={{ backgroundColor: menuContext?.backgroundColor, ml: 1 }}>
 							<ItemTypeIcon
 								fontSize="medium"
-								item={{ systemType: menuContext?.contentType.type ?? '', mimeType: '' } as SandboxItem}
+								item={{ systemType: menuContext?.contentType.type ?? '', mimeType: '' } as ContentItem}
```

Comment:
The `ItemDisplay` family of components like `ItemTypeIcon` should also declare `item: Pick<ContentItem, 'mimeType' | 'systemType' | etc>` to make them more flexible